packages/client/tests/functional/tracing/tests.ts

```diff
@@ -84,308 +84,378 @@ afterAll(() => {
+
+  beforeEach(async () => {
+    await prisma.$connect()
+    inMemorySpanExporter.reset()
+  })
+
+  async function waitForSpanTree(expectedTree: Tree): Promise<void> {
+    await waitFor(async () => {
+      await processor.forceFlush()
+      const spans = inMemorySpanExporter.getFinishedSpans()
+      const rootSpan = spans.find((span) => !span.parentSpanId) as ReadableSpan
+      const tree = buildTree(rootSpan, spans)
+
+      expect(tree).toEqual(expectedTree)
     })
+  }
 
-    async function waitForSpanTree(expectedTree: Tree): Promise<void> {
-      await waitFor(async () => {
-        await processor.forceFlush()
-        const spans = inMemorySpanExporter.getFinishedSpans()
-        const rootSpan = spans.find((span) => !span.parentSpanId) as ReadableSpan
-        const tree = buildTree(rootSpan, spans)
+  enum QueryChildSpans {
+    ArgsAndResult,
+    ArgsOnly,
+    None,
+  }
 
-        expect(tree).toEqual(expectedTree)
-      })
+  function dbQuery(statement: string, driverAdapterChildSpans = QueryChildSpans.ArgsAndResult): Tree {
+    const span = {
+      name: 'prisma:engine:db_query',
+      attributes: {
+        'db.statement': statement,
+      },
     }
 
-    function dbQuery(statement: string, hasDriverAdapterResultSpan = true): Tree {
-      const span = {
-        name: 'prisma:engine:db_query',
-        attributes: {
-          'db.statement': statement,
-        },
-      }
+    // extra children spans for driver adapters, except some queries (BEGIN/COMMIT with `usePhantomQuery: true`)
+    if (clientMeta.driverAdapter && driverAdapterChildSpans !== QueryChildSpans.None) {
+      const children = [] as Tree[]
 
-      // extra children spans for driver adapters
-      if (clientMeta.driverAdapter) {
-        const children = [] as Tree[]
+      children.push({
+        name: 'js:query:args',
+      })
 
-        // args span always exists for any query
+      // result span only exists for returning queries
+      if (driverAdapterChildSpans !== QueryChildSpans.ArgsOnly) {
         children.push({
-          name: 'js:query:args',
+          name: 'js:query:result',
         })
+      }
 
-        // result span only exists for returning queries
-        if (hasDriverAdapterResultSpan === true) {
-          children.push({
-            name: 'js:query:result',
-          })
-        }
+      children.push({
+        name: 'js:query:sql',
+        attributes: {
+          'db.statement': statement,
+        },
+      })
 
-        // sql span always exists for any query
-        children.push({
-          name: 'js:query:sql',
-          attributes: {
-            'db.statement': statement,
-          },
-        })
+      span['children'] = children
+    }
 
-        span['children'] = children
-      }
+    return span
+  }
 
-      return span
+  function txBegin() {
+    if (usesSyntheticTxQueries) {
+      return dbQuery('-- Implicit "BEGIN" query via underlying driver', QueryChildSpans.None)
```

Comment:
`expect.stringContaining('BEGIN')` would've covered both so only the second argument technically needs to be different but I decided to be explicit so that we can see the tests failing if the behaviour changes.